orbit/pkg/luks/luks.go

```diff
@@ -47,3 +50,22 @@ func extractJSON(input []byte) ([]byte, error) {
 	}
 	return []byte(match), nil
 }
+
+func isInstalled(toolName string) bool { //nolint:unused
```

Comment:
Move the `unused`s to `luks_linux.go`?

Reply:
The reason why I moved it here from luks_linux.go was because it was not Linux specific - but I guess we can do that refactor if/when we add LUKS support for other platforms.

Reply:
Done in dbd6a421f2ea489f967c61809d15c0151d46adeb